x-pack/plugins/reporting/server/lib/layouts/preserve_layout.ts

```diff
@@ -12,12 +12,13 @@ import {
   LayoutTypes,
   PageSizeParams,
   Size,
+  LayoutInstance,
 } from './';
 
 // We use a zoom of two to bump up the resolution of the screenshot a bit.
 const ZOOM: number = 2;
```

Comment:
Why has this never jumped out at me... we really should make this configurable as it's incredibly RAM inefficient.

Let's do that in a follow up at some point